drivers/sensors/Kconfig

```
@@ -561,6 +561,47 @@ config MCP9844_I2C_FREQUENCY
 	range 1 400000
 	depends on SENSORS_MCP9844
 
+config SENSORS_MS5611
+	bool "MS5611 Barometric Pressure Sensor support"
+	default n
+	select I2C
```

```suggestion
	select I2C
```
I think that I2C selection is done in `MS5611_I2C` case